From 37acfb15fc6a6b5328a2445e23dbac483dd30867 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 3 May 2026 22:28:41 +0800
Subject: [PATCH 550/820] AOSCOS: fbcon: add function declaration for
 fbcon_rotate_font_utf()

Fix build failure when built with CONFIG_WERROR.

Fixes: "BACKPORT: FROMEXT: cjktty-6.16.patch"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 drivers/video/fbdev/core/fbcon_rotate.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/fbdev/core/fbcon_rotate.h b/drivers/video/fbdev/core/fbcon_rotate.h
index 8cb019e8a9c0f..5026dadff8ed3 100644
--- a/drivers/video/fbdev/core/fbcon_rotate.h
+++ b/drivers/video/fbdev/core/fbcon_rotate.h
@@ -91,6 +91,7 @@ static inline void rotate_ccw(const char *in, char *out, u32 width, u32 height)
 }
 
 int fbcon_rotate_font(struct fb_info *info, struct vc_data *vc);
+int fbcon_rotate_font_utf(struct fb_info *info, struct vc_data *vc);
 
 #if defined(CONFIG_FRAMEBUFFER_CONSOLE_ROTATION)
 void fbcon_set_bitops_cw(struct fbcon_par *par);
-- 
2.52.0

